e2e/bitrise.yml

```diff
@@ -0,0 +1,260 @@
+  #      - OUTPUT_DIR: $BITRISE_DEPLOY_DIR
+  #      - BITRISE_APP_DIR_PATH_EXPECTED: "$BITRISE_DEPLOY_DIR/SampleProjectReactNative.app"
+  #      - BITRISE_APP_DIR_PATH_LIST_EXPECTED: "$BITRISE_DEPLOY_DIR/SampleProjectReactNative.app"
+  #      - CODE_SIGNING_ALLOWED: "no"
+  #    after_run:
+  #      - _common
+  #
+  #  test_react_native_workspace:
+  #    envs:
+  #      - XCODEBUILD_OPTIONS: -UseModernBuildSystem=NO
+  #      - SAMPLE_APP_URL: https://github.com/bitrise-samples/sample-project-react-native-xcworkspace.git
+  #      - BRANCH: master
+  #      - BITRISE_PROJECT_PATH: ios/SampleProjectReactNative.xcworkspace
+  #      - BITRISE_SCHEME: SampleProjectReactNative
+  #      - SIMULATOR_OS_VERSION: "latest"
+  #      - OUTPUT_TOOL: xcpretty
+  #      - OUTPUT_DIR: $BITRISE_DEPLOY_DIR
+  #      - BITRISE_APP_DIR_PATH_EXPECTED: "$BITRISE_DEPLOY_DIR/SampleProjectReactNative.app"
+  #      - BITRISE_APP_DIR_PATH_LIST_EXPECTED: "$BITRISE_DEPLOY_DIR/SampleProjectReactNative.app"
+  #      - CODE_SIGNING_ALLOWED: "no"
+  #    after_run:
+  #      - _common
+  #
+  #  test_react_native_custom_target_build_dir:
+  #    envs:
+  #      - XCODEBUILD_OPTIONS: -UseModernBuildSystem=NO
+  #      - SAMPLE_APP_URL: https://github.com/bitrise-samples/sample-project-react-native.git
+  #      - BRANCH: custom_target_build_dir
+  #      - BITRISE_PROJECT_PATH: ios/SampleProjectReactNative.xcodeproj
+  #      - BITRISE_SCHEME: SampleProjectReactNative
+  #      - SIMULATOR_OS_VERSION: "latest"
+  #      - OUTPUT_TOOL: xcpretty
+  #      - OUTPUT_DIR: $BITRISE_DEPLOY_DIR
+  #      - BITRISE_APP_DIR_PATH_EXPECTED: "$BITRISE_DEPLOY_DIR/SampleProjectReactNative.app"
+  #      - BITRISE_APP_DIR_PATH_LIST_EXPECTED: "$BITRISE_DEPLOY_DIR/SampleProjectReactNative.app"
+  #      - CODE_SIGNING_ALLOWED: "no"
+  #    after_run:
+  #      - _common
+
+  #  test_swiftpm:
+  #    envs:
+  #      - XCODEBUILD_OPTIONS:
+  #      - SAMPLE_APP_URL: https://github.com/bitrise-io/sample-apps-ios-swiftpm.git
+  #      - BRANCH: master
+  #      - BITRISE_PROJECT_PATH: sample-swiftpm.xcodeproj
+  #      - BITRISE_SCHEME: "sample swiftpm"
+  #      - OUTPUT_TOOL: xcodebuild
+  #      - SIMULATOR_OS_VERSION: "latest"
+  #      - OUTPUT_DIR: $BITRISE_DEPLOY_DIR
+  #      - BITRISE_APP_DIR_PATH_EXPECTED: $BITRISE_DEPLOY_DIR/sample-swiftpm.app
+  #      - BITRISE_APP_DIR_PATH_LIST_EXPECTED: $BITRISE_DEPLOY_DIR/sample-swiftpm.app
+  #      - CODE_SIGNING_ALLOWED: "no"
+  #    after_run:
+  #      - _common
+
+  _common:
+    steps:
+      - script:
+          inputs:
+            - content: |-
+                #!/bin/bash
+                set -ex
+                rm -rf "$ORIG_BITRISE_SOURCE_DIR/_tmp"
+                mkdir -p "$ORIG_BITRISE_SOURCE_DIR/_tmp"
+      - change-workdir:
+          title: Switch working dir to _tmp dir
+          run_if: "true"
+          inputs:
+            - path: "$ORIG_BITRISE_SOURCE_DIR/_tmp"
+            - is_create_path: true
+      - script:
```

Comment:
This custom script can be replaced with https://github.com/bitrise-steplib/bitrise-step-simple-git-clone

It's poorly documented (🙈 ), but you can see usage examples of this step in other repos.

Reply:
Nice, definitely will prefer to use a Step if it's available 👍

- [ ] Switch to Step for git cloning